contributors/guide/release-notes.md

```diff
@@ -16,20 +16,14 @@ For pull requests with a release note:
     Your release note here
     ```
 
-For pull requests that require additional action from users switching to the new release, include the string "action required" (case insensitive) in the release note:
+For pull requests that require additional action from users switching to the new release, write your Release Note as usual and add the `release-note-action-required` label to the PR.
```

Comment:
If the words "action required" are not mentioned in the release note, the bot will automatically add the `release-note` label.

If the words "action required" are present in the release note, the bot will automatically add the `release-note-action-required` label.

I think writing the words "action required" are still relevant. If we add the release note as usual and then add then use the command `/release-note-action-required`, it will have two release note labels: `release-note-none` and `release-note-action-required`.

Reply:
Oh, I just checked: https://prow.k8s.io/command-help. There is no `/release-note-action-required` command. Looks the words "action required" are really needed.